core/embed/bootloader/main.c

```diff
@@ -124,34 +130,34 @@ static secbool bootloader_usb_loop(const vendor_header *const vhdr,
         break;
       case 5:  // WipeDevice
         ui_fadeout();
-        ui_screen_wipe_confirm();
-        ui_fadein();
-        int response = ui_user_input(INPUT_CONFIRM | INPUT_CANCEL);
+        uint32_t response = ui_screen_wipe_confirm();
         if (INPUT_CANCEL == response) {
           ui_fadeout();
-          ui_screen_firmware_info(vhdr, hdr);
+          screen_connect();
           ui_fadein();
           send_user_abort(USB_IFACE_NUM, "Wipe cancelled");
-          break;
+          usb_stop();
+          usb_deinit();
```

Comment:
This was requested by product @Hannsek . It allows get back to the menu when action from host is cancelled, otherwise user needs to reconnect the device.

Used will be disconnected and will need to connect again via the "install firmware" button

Reply:
Seems to be working fine. There needs to be added delay between sending abort and deiniting usb so that there is no error. This was done for the install firmware, but not for the wipe, so i added that here: https://github.com/trezor/trezor-firmware/commit/0dbe1f378a31aeb30443a637a06bb599f66e5a2a . 

Suite also handles this ok, but there is some room for improvement: it stays in the device settings page, saying that the device is not connected. Might be better to redirect to the initial screen when this happens.